frontend/javascripts/admin/organization/organization_cards.tsx

```diff
@@ -194,9 +194,9 @@ export function PlanDashboardCard({
     organization.includedStorageBytes === Number.POSITIVE_INFINITY ? "∞" : includedStorageLabel;
 
   const usedStorageLabel =
-    organization.pricingPlan === PricingPlanEnum.Basic
-      ? `${(organization.usedStorageBytes / 10 ** 9).toFixed(1)}`
-      : `${(organization.usedStorageBytes / 10 ** 12).toFixed(1)}`;
+    organization.pricingPlan === PricingPlanEnum.Basic || organization.usedStorageBytes < 10 ** 11 //=0.1TB
```

Comment:
While reading this code, I remembered that we already have a utility function for formatting bytes. I think, this would simplify the existing code quite a bit. I imagine something like this:

```ts
  const includedStorageLabel =
    organization.includedStorageBytes === Number.POSITIVE_INFINITY
      ? "∞"
      : formatCountToDataAmountUnit(organization.includedStorageBytes);
  const usedStorageLabel = formatCountToDataAmountUnit(organization.usedStorageBytes);
```

That way, one doesn't need to tie the formatting logic to the organization plan, either.

Reply:
Oh, nice!
During testing I noticed that the formatting is different than I would have implemented it. In the case of this picture, I would have rendered "0.1TB/5TB" or something, because there would be no break in `GB`. But I think it's still readable, so should I just leave it at your suggested code?
![Screenshot from 2023-05-09 19-05-16](https://github.com/scalableminds/webknossos/assets/57364771/08a41198-62c2-4a9c-923c-2e910e8db93e)